ivy_tests/test_ivy/test_frontends/test_torch/test_func_wrapper.py

```diff
@@ -0,0 +1,51 @@
+# global
+from hypothesis import given
+
+# local
+import ivy
+import ivy_tests.test_ivy.helpers as helpers
+from ivy.functional.frontends.torch.func_wrapper import (
+    inputs_to_ivy_arrays,
+    outputs_to_frontend_arrays,
+    to_ivy_arrays_and_back,
+)
+from ivy.functional.frontends.torch.tensor import Tensor
+
+
+def _fn(x):
+    return x
+
+
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("valid")),
+)
+def test_inputs_to_ivy_arrays(dtype_and_x):
+    x_dtype, x = dtype_and_x
+    input = Tensor(x[0])
+    input.ivy_array = ivy.array(x[0], dtype=x_dtype[0])
+    output = inputs_to_ivy_arrays(_fn)(input)
+    assert isinstance(output, ivy.Array)
+    assert str(input.dtype) == str(output.dtype)
+
```

Comment:
Add `assert input._ivy_array == output`. 
Also add an ivy array and native_array input check.